internal/controller/http/v2/devicemanagement_test.go

```diff
@@ -40,6 +40,23 @@ func deviceManagementTest(t *testing.T) (*MockDeviceManagementFeature, *gin.Engi
 func TestGetFeatures(t *testing.T) {
 	t.Parallel()
 
+	featuresInput := dto.Features{
```

Comment:
for v2 the goal was to have one struct -- dtov2 for both input and output right?

Reply:
updated